test/curve25519_selftest.c

```diff
@@ -390,8 +390,13 @@ void edp_DualPointMultiply(
 void print_words(IN const char *txt, IN const U_WORD *data, IN U32 size)
 {
     U32 i;
-    printf("%s0x%08X", txt, (unsigned int)(*data++));
-    for (i = 1; i < size; i++) printf(",0x%08X", (unsigned int)(*data++));
+#ifdef USE_ASM_LIB
+    printf("%s0x%08llX", txt, *data++);
+    for (i = 1; i < size; i++) printf(",0x%08llX", *data++);
+#else
```

Comment:
1. I rather using WORDSIZE_64 instead of USE_ASM_LIB
2. %08llX should be changed to %016llX

3. Thanks

Reply:
Done. Thank you.